lib/app/item.ex

```diff
@@ -2,23 +2,26 @@ defmodule App.Item do
   use Ecto.Schema
   import Ecto.Changeset
   import Ecto.Query
-  alias App.{Repo, Tag, ItemTag, Person}
+  alias App.{Repo, Tag, ListItem, ItemTag, Person}
+  alias App.List, as: L
   alias __MODULE__
 
   schema "items" do
     field :status, :integer
     field :text, :string
+    field :item_lists, {:array, :string}, virtual: true
 
     belongs_to :people, Person, references: :person_id, foreign_key: :person_id
```

Comment:
The question of "ownership" of an `item` will be revisited when we explore permissions later. 💭 
This model implies that an `item` can only have **one** "owner". 1️⃣ 
Much the way real-world property can have multiple owners, `items` will too. 🏠